numpy/core/src/common/npy_config.h

```diff
@@ -2,42 +2,31 @@
 #define NUMPY_CORE_SRC_COMMON_NPY_CONFIG_H_
 
 #include "config.h"
-#include "npy_cpu_features.h"
-#include "npy_cpu_dispatch.h"
 #include "numpy/numpyconfig.h"
-#include "numpy/npy_cpu.h"
+#include "numpy/utils.h"
 #include "numpy/npy_os.h"
 
 /* blocklist */
 
-/* Disable broken Sun Workshop Pro math functions */
-#ifdef __SUNPRO_C
-
-#undef HAVE_ATAN2
-#undef HAVE_ATAN2F
-#undef HAVE_ATAN2L
-
-#endif
-
 /* Disable broken functions on z/OS */
 #if defined (__MVS__)
 
-#undef HAVE_POWF
-#undef HAVE_EXPF
+#define BLOCK_POWF
+#define BLOCK_EXPF
```

Comment:
If we're renaming these variables already, should we take the opportunity to better namespace them? I.e. `NPY_BLOCK_EXPF`. AFAICT that's better practice than having such numpy-specific things in the global macro namespace.

Reply:
Done.